lib/Migration/Version010104Date20250102000000.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\PhotoFrames\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+/**
+ * Add style_fill and style_background_color columns to photo_frames_frames table
+ */
+class Version010104Date20250102000000 extends SimpleMigrationStep
+{
+
+  /**
+   * @param IOutput $output
+   * @param Closure(): ISchemaWrapper $schemaClosure
+   * @param array $options
+   */
+  public function preSchemaChange(IOutput $output, Closure $schemaClosure, array $options): void
+  {
+  }
+
+  /**
+   * @param IOutput $output
+   * @param Closure(): ISchemaWrapper $schemaClosure
+   * @param array $options
+   * @return null|ISchemaWrapper
+   */
+  public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper
+  {
+    /** @var ISchemaWrapper $schema */
+    $schema = $schemaClosure();
+
+    $table = $schema->getTable('photo_frames_frames');
+
+    if (!$table->hasColumn('style_fill')) {
+      $table->addColumn('style_fill', Types::BOOLEAN, ['notnull' => false, 'default' => false]);
```

Comment:
As mentioned in the comment I wrote earlier today, I believe this should be a string `cropMode` (or something like that). It could then default to "contain", and you could add a "cover" setting.

It will leave me the space to add more advanced settings in the future.